Assets/Scripts/Interface/CarStats.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Cameras;
+using DataTypes;
+
+public class CarStats : MonoBehaviour
+{
+    public Transform freeCamera;
+    public TMPro.TMP_Text speedText;
+    public GameObject panel;
+
+    FreeCamera _freeCam;
+    CarBehaviour _car;
+    bool _visible = false;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        _freeCam = freeCamera.GetComponent<FreeCamera>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (_freeCam._following && !_visible)
+        {
+            Debug.Log("now");
+            _visible = true;
+            panel.SetActive(true);
+            _car = _freeCam._targetCar.GetComponent<CarBehaviour>();
```

Comment:
Wichtig: Wenn #85 gemerged ist, gibt es nur noch `LinkedBehaviour`. Also sollte man:
1. Statt dem behaviour das Car objekt direkt speichern und
2. Muss `LinkedBehaviour.data` manuell zu `Car` gecasted werden (8e0ac169c43f4e4d812997b9935155b56b33128d).

Reply:
Also sind die changes im develop?
Das mit dem Speichern des behaviours war halt weil nur ein carBehaviour auf dem car object war.
Zum Bug mit dem Laden der Szene, also eine andere test-Szene mit allem außer dem simulationcontroller (und auch ohne Straßen und Co deswegen) könnte einwandfrei geladen werden. Also anscheinend ist etwas (weiß nicht ob es an Sachen wie Debug.Log liegen könnte) mit unserem Code, der das game crasht.

Reply:
Nein, die Änderungen sind noch nicht in develop (#85 ist noch nicht reviewed), wollte nur schon drauf hinweisen.
Wegen dem Bug: Merge am besten mal develop hier rein, inzwischen gibts ja keinen simulationcontroller mehr, sondern nur nen FileLoader. Wüsste nicht, inwiefern das helfen könnte, aber man sollte es probieren.